client-base/src/main/webapp/html/javascript/testFeatures.js

```diff
@@ -0,0 +1,48 @@
+var resetBtnEle = $("#resetKnet")
+
+// function gets all inputs in the search form and check if value is present
+// if value is present it activates the reset button
+function ActivateResetButton(){
```

Comment:
naming conventions here too (function names go uncapitalised in Js, class names only are capitalised).